tests/Makefile

```diff
@@ -0,0 +1,23 @@
+include ../embedded-uart-common/test-config/base_config.inc
+sps_driver_dir := ../
+include ${sps_driver_dir}/sps30-uart/default_config.inc
+
+sps30_test_binaries := sps30-test-uart
+
+linux_uart_sources := ${sensirion_common_dir}/sample-implementations/linux/sensirion_uart_implementation.c
```

Comment:
In analogy to the i2c drivers, this should not be needed, only `uart_sources?=../embedded-uart-common/sensirion_uart_implementation.c` (with this default). Then the environment can provide it at build time: i.e. `uart_sources=../embedded-uart-common/sample-implementations/linux/sensirion_uart_implementation.c make -C tests test`

Reply:
Ah yeah got it now :) Changed it.